ui/lang/index.json

```diff
@@ -218,5 +218,10 @@
   {
     "isoName": "zh-TW",
     "nativeName": "中文（繁體）"
+  },
+  ,
```

Comment:
There's an extra comma.

Reply:
I think more important is that the language file is missing :)